src/docs/develop/cli.md

```diff
@@ -7,84 +7,122 @@ alt="GIF of the CLI in Action"
+- [Scoop](https://scoop.sh) for Windows as a native `.exe` binary
+- Direct Shell Script
+- Downloading pre-built binaries
+- Building from source
 
-### NPM (Linux, macOS, Windows)
+### Homebrew (macOS)
+
+In a Terminal, enter the following command:
 
 ```bash
-npm i -g @railway/cli
+brew install railway
 ```
 
-You need to have => 16.x version of [Node.js](https://nodejs.org/en/) installed.
+### npm (macOS, Linux, Windows)
 
-### Shell Script (Linux, macOS, Windows: Git Bash)
+In a Terminal, enter the following command:
 
 ```bash
-curl -fsSL cli.new | sh
+npm i -g @railway/cli
 ```
 
-### Homebrew (Linux, macOS)
+This requires version =>16 of [Node.js](https://nodejs.org/en/).
+
+### Shell Script (macOS, Linux, Windows)
+
+In a Terminal, enter the following command:
 
 ```bash
-brew install railway
+bash <(curl -fsSL cli.new)
 ```
 
+On Windows, you should use [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install)
+with a Bash shell.
+
 ### Scoop (Windows)
 
-Use this method if you prefer to interact with Railway using a native Windows binary. In a PowerShell terminal enter the following command:
+In a PowerShell terminal, enter the following command:
 
 ```ps1
 scoop install railway
 ```
 
-For additional documentation on Scoop, see [here](https://scoop.sh/).
+This installs a native Windows binary (`.exe`). To learn more about Scoop,
+see [https://scoop.sh/](https://scoop.sh/).
 
-### Source
+### Pre-built Binaries
 
-You can also download the [prebuilt binary directly](https://github.com/railwayapp/cli/releases/latest) or [build from source](https://github.com/railwayapp/cli#from-source).
+We publish [pre-built binaries](https://github.com/railwayapp/cli/releases/latest)
+on our [GitHub repository](https://github.com/railwayapp/cli) that you can
+download and use directly.
 
-## Authentication
+### From Source
 
-### Interactive Login
+The Railway CLI is an open source project on [GitHub](https://github.com/railwayapp/cli).
+You can build a binary [from source](https://github.com/railwayapp/cli#from-source)
+if you wish.
 
-Login to your Railway account. This command opens a browser tab which authenticates into an existing Railway session.
+## Authentication
+
+Before you can use the Railway CLI, you must authenticate the CLI to your Railway account:
 
 ```bash
 railway login
 ```
 
-If there are any issues with browser-based login you can use the `--browserless` flag to authenticate.
+This command opens a new tab in your default browser to the [https://railway.app](https://railway.app)
+authentication page. Follow the instructions to complete the authentication process.
+
+### Manual Login
 
-This will print a URL and a pairing code to the terminal which you can use to authenticate (useful for SSH sessions).
+You can also authenticate manually using a Pairing Code. This can be useful if
+you're authenticating the CLI inside an environment without a browser (e.g. SSH
+sessions).
```

Comment:
No review comment but just pointing out what I like about this PR. This is clear, explains benefit, and an improvement on the curt nature of the copy.